lang_GENERIC_base/AST_generic.ml

```diff
@@ -749,6 +749,8 @@ and stmt = {
    * Indeed, the main use of those 'tok' is to accurately report a match range
    * in semgrep.
   *)
+  (* used in semgrep to skip some AST matching *)
+  mutable s_bf: Bloom_filter.t option [@equal fun _a _b -> true];
```

Comment:
A more descriptive field name like `s_bloom_filter` would be nice.

Reply:
I didn't know we could tweak the derived equality function with `[@equal fun _a _b -> true]`. I see you added one on the `s_backrefs` field as well, which is great.